alacritty/Cargo.toml

```diff
@@ -13,6 +13,7 @@ rust-version = "1.70.0"
 path = "../alacritty_terminal"
 version = "0.20.0-dev"
 default-features = false
```

Comment:
Do you know why we're disabling default features here? I don't think there are any.

Reply:
I don't, probably just because we do so for everything.